fink_science/ztf/superluminous/processor.py

```diff
@@ -0,0 +1,146 @@
+# Copyright 2020-2023 AstroLab Software
+# Author: Etienne Russeil
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+from line_profiler import profile
+from fink_science import __file__
+from pyspark.sql.functions import pandas_udf, PandasUDFType
+from pyspark.sql.types import DoubleType
+from fink_science.tester import spark_unit_tests
+import numpy as np
+import pandas as pd
+import fink_science.ztf.superluminous.slsn_classifier as slsn
+from fink_science.ztf.superluminous.kernel import classifier_path
+import joblib
+import os
+
+
+@pandas_udf(DoubleType())
+@profile
+def superluminous_score(
+    cjd: pd.Series,
+    cfid: pd.Series,
+    cmagpsf: pd.Series,
+    csigmapsf: pd.Series,
+    candidate: pd.Series,
```

Comment:
candidate is a big field containing many other fields. If you only need `distnr`, you can just put `distnr`